bindings/rust/bench/certs/generate_certs.sh

```diff
@@ -21,6 +21,7 @@ cert-gen () {
 
     key_family=$1
     key_size=$2
+    dir_name=$3
```

Comment:
I think it might be time for a usage function, your script is getting really complex: https://stackoverflow.com/a/687820.

Reply:
Since the function is used only for the script (sort of a private function? but bash doesn't have scopes like that), I think a separate usage function would add too much complexity for not enough gain. I did forget to add a comment about the new `dir_name` argument, so I'll add that.